RELEASING.md

```diff
@@ -156,7 +156,11 @@ Here are the steps:
 1. Since you have 15 minutes free, use that time to update https://github.com/adoptium/adoptium.net/blob/master/src/handlebars/support.handlebars which is the source of  https://adoptium.net/support.html and (if required) the supported platforms table at https://github.com/adoptium/adoptium.net/blob/master/src/handlebars/supported_platforms.handlebars which is the source of https://adoptium.net/supported_platforms.html, and also update https://adoptium.net/release_notes.html ([Sample change](https://github.com/adoptium/adoptium.net/pull/675/commits/563d8e2f0d9d26500a7e8d9eca61b491f73f1f37)
-1. [Mac only] Once the binaries are available on the website you need to run the [homebrew cask updater](https://github.com/AdoptOpenJDK/homebrew-openjdk/wiki/Running-the-cask-updater) which will create a pull request [here](https://github.com/AdoptOpenJDK/homebrew-openjdk/pulls). Normally George approves these but in principle as long as the CI passes, they should be good to approve. You don't need to wait around and merge the PR's because the Mergify bot will automatically do this for you as long as somebody has approved it.
+1. [Mac only] Once the binaries are available on the website you need to update the Homebrew casks. There are 3 casks in total ([`temurin`](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/temurin.rb) which always serves the latest version, [`temurin8`](https://github.com/Homebrew/homebrew-cask-versions/blob/master/Casks/temurin8.rb) and [`temurin11`](https://github.com/Homebrew/homebrew-cask-versions/blob/master/Casks/temurin11.rb)). An example PR can be found [here](https://github.com/Homebrew/homebrew-cask/commit/4565865e9d7c3d3018ee8aa67803ea68c54dde86). Shamsums can be easilty copied from the website. The separate pull request is required for each version you update. If in doubt reach out to George (gdams) as he's a maintainer.
+1. [Linux only] Once the binaries are available on the website you can begin updating the specfiles for the RPM/DEB files. There are 3 different type of linux installer ([debian](https://github.com/adoptium/installer/tree/master/linuxNew/jdk/debian/src/main/packaging/temurin), [redhat](https://github.com/adoptium/installer/tree/master/linuxNew/jdk/redhat/src/main/packaging/temurin) and [suse](https://github.com/adoptium/installer/tree/master/linuxNew/jdk/suse/src/main/packaging/temurin)) and all 3 need updating. Once these files have been updated the [adoptium-packages-linux-pipeline](https://ci.adoptopenjdk.net/job/adoptium-packages-linux-pipeline/) job needs to be kicked off making sure to tick the `UPLOAD` parameter. It is suggested that you run the job with the `dry-run` parameter first to ensure that your parameters are correctly defined.
```

Comment:
Ref my initial comment:

> Presumably it is valid to run the packages job without UPLOAD initially to use as a dry-run similar to the main publish job to verify that no errors occur?

I don't think the job currently has a parameter called `dry-run` (but we should possibly change the job to do so) so this needs to say "It is suggested that you run the job with `UPLOAD` unchecked first"